pkg/sortition/internal/local/local.go

```diff
@@ -1,4 +1,4 @@
-package sortition
+package local
```

Comment:
Good point. I did the file rename in https://github.com/keep-network/keep-core/pull/3077/commits/3c68e2cdd87eb4122a7ae070323650e9d848f2a6. The struct rename is a bit more complicated because we import `"github.com/keep-network/keep-core/pkg/chain"` in that file and can't have a structure named `chain`. I experimented with aliasing the package but I think it brought more obfuscation than clarity.

Reply:
Maybe we should consider exporting the channel struct and have `local.Chain`? That seems to make sense as we have `local.Connect` function that returns that struct outside.